### PR TITLE
cowsay: add listing command

### PR DESCRIPTION
**Pull request by smileytechguy**

### CODE REVIEW

File: pages/common/cowsay.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `cowsay "Hello world!"`
 
+- List all availiable characters:
```
Comment (agnivade) [inline pages/common/cowsay.md]: typo => availiable

**Comment by agnivade** [comment]
cc /@jeeftor

**Comment by waldyrious** [comment]
For context, an example with a similar goal was added in #1266 and later removed in #1323 because it was too complex for tldr-pages standards. I'm glad there's an easier way to achieve this :)

I suppose the second line of the description is now redundant and could be removed. What do you think, @smileytechguy?

**Comment by agnivade** [comment]
Agree.

File: pages/common/cowsay.md (hunk 1 above)
Comment (smileytechguy) [inline pages/common/cowsay.md]: oops, thanks

**Comment by agnivade** [comment]
Hey @smileytechguy - Like @waldyrious mentioned, lets remove the 2nd line of the main description.

**Comment by waldyrious** [comment]
> Hey @smileytechguy - Like @waldyrious mentioned, lets remove the 2nd line of the main description.

I guess we can do that ourselves :)

**Comment by waldyrious** [comment]
@smileytechguy: For future reference, next time please create a separate branch for the PRs, to avoid possible merge conflicts (and so that, in case maintainers edit the PR, they don't edit your master branch). Cheers!

**Comment by waldyrious** [comment]
I'll merge this on the command line, to cleanup the commits a bit

**Comment by smileytechguy** [comment]
Awesome, thanks

**Pull request merged**